src/EssentialsPE/Loader.php

```diff
@@ -103,8 +103,7 @@ public function onEnable(): void{
         if(!is_dir($this->getDataFolder())){
             mkdir($this->getDataFolder());
         }
-        
-		$this->getLogger()->info(TextFormat::YELLOW . "Loading...");
```

Comment:
This change appears not to pertain to updating the API, nor does the one below.

Reply:
I didn't get that quite right. It was removed as part of bit.ly/pmcolors

Reply:
I asked him to do this change.